DICOM/CustomAnalysisRules.ruleset

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="FHIR Analysis Rules" Description="The code analysis rules used for the FHIR Tools for Anonymization C# source projects." ToolsVersion="14.0">
```

Comment:
Shall we use the new brand name Tools-for-Health-Data-Anonymization?

Reply:
I modified it in #133  which could be merged after this release.